src/Microsoft.Health.Fhir.SqlServer/Features/Storage/Registry/SqlServerStatusRegistry.cs

```diff
@@ -0,0 +1,115 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Features.Search.Registry;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer.Features.Storage;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage.Registry
+{
+    internal class SqlServerStatusRegistry : ISearchParameterRegistry
```

Comment:
nit: Should we rename this to SqlServerStatusRegistryDataStore for consistency?

Reply:
I followed the naming of the CosmosDB file `CosmosDbStatusRegistry.cs`. @brendankowitz thoughts on renaming both of these files?

Reply:
Makes sense

Reply:
Renamed in b38734b and 01b9a6a.